hello-world-master/source/components/scene/getrequesttask.brs

```diff
@@ -0,0 +1,47 @@
+sub init()
+	m.top.functionName = "getData"
+end sub
+
+
+sub getData()
+    json = executeGetRequest("https://my-json-server.typicode.com/bogdanterzea/pokemon-server/photos")
+    listRoot = CreateObject("roSGNode","ContentNode")
+    row = CreateObject("roSGNode","ContentNode")
+    row2 = CreateObject("roSGNode","ContentNode")
+    row.TITLE = "Pokemons"
+    for each item in json
+        rowChild = CreateObject("roSGNode","itemContentNode")
+        rowChild.id = item.id
+        rowChild.title = item.title
+        rowChild.url = item.url
+        rowChild.image_1080_url = item.image_1080_url
+        rowChild.description = item.description
+        rowChild.FHDPOSTERURL = item.url
+        row.appendChild(rowChild)
+    end for
+    
+    listRoot.appendChild(row)
+    m.top.itemContent = listRoot
+end sub
+
+function executeGetRequest(url) 
```

Comment:
Usually for functions you should mention the type of the parameters as you did here -> https://github.com/AndreiCostinOlaru/Roku/pull/4/files#diff-da73411e22e7e208029059bd5d0cbfd105f74b42204b75d9f7d8f8cfad57b55bR21
Also, this function doesn't return anything, so it can be a sub